spec/models/miq_group_spec.rb

```diff
@@ -458,4 +467,35 @@
       expect(group.user_count).to eq(2)
     end
   end
+
+  describe "#single_group_users?" do
+    it "returns false if all users in the group belong to an additional group" do
```

Comment:
`returns false if any users...` right?

If it makes sense, please reword that in a followup PR.